src/actor/Actor_test.ts

```diff
@@ -83,6 +89,35 @@ suite("Actor", () => {
     });
   });
 
+  test("re-traverses messages after hookup", async () => {
+    await new Promise(async resolve => {
+      let ignoringHookdown: HookdownCallback;
+
+      class LateActor extends Actor<"dummy"> {
+        onMessage() {
+          resolve();
+        }
+      }
+
+      class IgnoringActor extends Actor<"dummy"> {
+        onMessage() {
+          setTimeout(async () => {
+            const lateHookdown = await hookup("late", new LateActor());
+
+            hookdown = async () => {
+              await ignoringHookdown();
+              await lateHookdown();
+            };
+          }, 100);
```

Comment:
Why the timeout?

Reply:
To make sure all messages were traversed by all realms before we call hookup. Otherwise, I ran into the issue where, because of the IDB transactions, the messages would still be delivered without the fix. E.g. the test would not fail properly without the fix.